packages/config/packages/config/config/devices/0x0371/zw141.json

```diff
@@ -0,0 +1,503 @@
+{
+	"manufacturer": "Aeotec Limited",
+	"manufacturerId": "0x0371",
+	"label": "ZW141",
+	"description": "Nano Shutter",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x008d"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Retransmit",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"20": {
+			"label": "Motor behavior after powering on",
+			"description": "Set the motor behavior after powering on",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Stay the same position",
+					"value": 0
+				},
+				{
+					"label": "Move to 99%",
+					"value": 1
+				},
+				{
+					"label": "Move to 0%",
+					"value": 2
+				}
+			]
+		},
+		"22": {
+			"label": "Motor direction",
+			"description": "Set to toggle the motor running direction",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 2,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false
```

Comment:
This description suggests there are only two options. You're going to have to track the manual down and add the options. This is already set to allowManualEntry: false, so it is effectively unusable at the moment. Make sure you fix the maxValue once you identify the real values.

Reply:
Fixed according to  https://help.aeotec.com/support/solutions/articles/6000198854-nano-shutter-user-guide-